prose.md

```diff
@@ -51,169 +52,169 @@ grass, which leads to a crosswalk. There is a sign in front of the unpaved path
+1. To maintain a consistent style that emphasized our principles of
+   evidence-based teaching such as learning objectives and formative assessment;
+2. To demonstrate how the skills we teach in our workshops could be applied
    to real-life situations.
 
-While this was conceptually good in theory, this infrastructure design has two
+While this was conceptually good in theory, this infrastructure design has three
 significant drawbacks for contributors:
 
 1. Installation pains.
    <https://twitter.com/gvwilson/status/1380115181708189701?s=20> Having
    Jekyll, Python, and Make as dependencies means that people who want to build
-   these lessons need all three of these successfully installed and up-to-date on
-   their machines. This is especially frustrating for Windows users who have none
-   of these by default. Moreover, having all the tools living inside the
-   repository meant that lessons quickly become out of date as we improve our
-   infrastructure.
+   these lessons on their machines need all three of these successfully installed 
+   and up-to-date.
+   This is especially frustrating for Windows users who have none
+   of these by default.
+2. All the scripts live inside the repository. This means that lessons quickly
+   become out of date as they diverge from the improvements made upstream.
 2. Another drawback is that we have a lesson website wrapped around a static
    site generator (which in and of itself is a kind of desire path), this meant
    that it was easy to contribute to if you were familiar with how Jekyll
-   operated. Some would say Jekyll is well documented, but in my experience,
-   Jekyll is well, documented. There is often a moment of panic in a new
+   operated. There is often a moment of panic in a new
    contributor's eyes when you show them what the a lesson repository looks like.
    If you were not familiar with Jekyll, then it was unclear where to even start
-   if you were looking at the git repository because of all the exposed wiring
-   with no signs that clearly say "start here". This aspect has lead to several
-   lessons built in _slightly_ different ways.
-
-This brings me back to the cowpaths. Over the last few years, we have
-begun finding them from all across the lesson infrastructure popping up in
-separate places that appear through issues, pull requests and general
-frustration from contributors and maintainers alike. The thing about cowpaths,
+   if you were looking at the git repository because there were no clear sign
+   that marked the trail head. This lead several lessons to find their own
+   trail head that ended up built in different ways.
+
+Over the last few years, we have
+begun finding the trail head of these desire paths all across our lesson
+infrastructure.
+This manifested through our communication channels and frustration from contributors
+and maintainers alike. The thing about desire paths,
```

Comment:
I think we should avoid mixing these metaphors since trail heads are associated with official paths.